__tests__/list.spec.js

```diff
@@ -22,6 +22,17 @@ const questionListNoDefault = [
   }
 ];
 
+const Inquirer_Default_Separator = {type:"separator",line:'\u001b[2m──────────────\u001b[22m'};
+const Inquirer_Sample_Separator = {type:"separator",line:'\u001b[2mCustom Separator\u001b[22m'};
+const questionListWithSeparator = [
+  {
+    type: "list",
+    name: "country",
+    message: "The country where you live",
+    choices: ["USA", Inquirer_Default_Separator, "Germany", Inquirer_Sample_Separator, "China", "Israel"]
```

Comment:
better we use new Inquirer.Separator("") new Inquirer.Separator("Custom Separator") 
then if inquirer will change their default the test will fail.